packages/governance/src/types-ambient.js

```diff
@@ -61,7 +61,7 @@
  * Terms a contract must provide in order to be governed.
  *
  * @template {import('./contractGovernance/typedParamManager.js').ParamTypesMap} T Governed parameters of contract
- * @typedef {{ electionManager: Instance, governedParams: import('./contractGovernance/typedParamManager.js').ParamRecordsFromTypes<T & { Electorate: 'invitation' }> } } GovernanceTerms<T>
+ * @typedef {{ electionManager: import('@agoric/zoe/src/zoeService/utils.js').Instance<import('./contractGovernor').start>, governedParams: import('./contractGovernance/typedParamManager.js').ParamRecordsFromTypes<T & { Electorate: 'invitation' }> } } GovernanceTerms<T>
```

Comment:
I'm struggling to read this. linebreaks and indentation, please?

```js
/**
 * @typedef {{
 *   electionManager: import('@agoric/zoe/src/zoeService/utils.js').Instance<import('./contractGovernor').start>,
 *   governedParams: import('./contractGovernance/typedParamManager.js').ParamRecordsFromTypes<T & {
 *      Electorate: 'invitation',
 *      }>,
 *  }} GovernanceTerms<T>
 */
```

or perhaps it's worth an extra hop to the prose docs for `Instance`:

```js
/** @param CF @typedef {import('@agoric/zoe/src/zoeService/utils.js').Instance<CF>} Instance<CF> */
/** @param T @typedef {import('./contractGovernance/typedParamManager.js').ParamRecordsFromTypes<T>} ParamRecordsFromTypes<T> */
/**
 * @typedef {{
 *   electionManager: Instance<import('./contractGovernor').start>,
 *   governedParams: ParamRecordsFromTypes<T & { Electorate: 'invitation' }>,
 *  }} GovernanceTerms<T>
 */
```

(I'm reviewing by commit, so the github suggest widget doesn't work.)

Reply:
Hm... must the `electionManager` be an instance of `contractGovernor`? I suppose we rely on it being interface-compatible... or: we haven't defined any more narrow interface.

Reply:
Or do we? Does the client ever make calls to the `electionManager`? Or would `Instance<unknown>` work?

That is: is it just there for legibility?

Reply:
Yep, it's what `getContractGovernor` returns. With unknown, here's what the type would be:
<img width="504" alt="Screenshot 2022-12-20 at 3 31 21 PM" src="https://user-images.githubusercontent.com/21505/208785887-74926676-269d-4210-ad37-024640d65dab.png">

It doesn't cause any type errors, but that's because nothing yet calls `getContractGovernor`. So I'd say this parameter is for legibility and correctness.